### PR TITLE
Test for export/import device needed updating; service bug still exis…

### PR DESCRIPTION
**Pull request by drwill-ms**
Welp, we accidentally introduced the ParentScopes property in ExportImportDevice but the service is still not exporting that value. I've updated the test in anticipation of it being fixed.

### CODE REVIEW

File: iothub/service/src/ExportImportDevice.cs [hunk 1]
```diff
@@ -21,19 +21,18 @@ public sealed class ExportImportDevice
         private string _twinETag;
 
         /// <summary>
-        /// Property container
+        /// Tyep definition for the <see cref="Properties"/> property.
```
Comment (timtay-microsoft) [inline iothub/service/src/ExportImportDevice.cs:24]:
```suggestion
        /// Type definition for the <see cref="Properties"/> property.
```

File: e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs [hunk 2]
```diff
@@ -47,14 +47,15 @@ public class RegistryManagerExportDevicesTests : E2EMsTestBase
         [Timeout(120000)]
         [DoNotParallelize]
         [DataRow(StorageAuthenticationType.KeyBased, false)]
-        [DataRow(StorageAuthenticationType.IdentityBased, false)]
-        [DataRow(StorageAuthenticationType.IdentityBased, true)]
+        //[DataRow(StorageAuthenticationType.IdentityBased, false)]
```
Comment (azabbasi) [inline e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs:50]: Can you elaborate why these are commented out?

**Review by timtay-microsoft — APPROVED** [review]

File: e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs (hunk 2 above)
Comment (drwill-ms) [reply e2e/test/iothub/service/RegistryManagerExportDevicesTests.cs:50]: Accidentally included in the PR. Thanks for catching. They were making it awkward to debug so I (meant to) temporarily disable them.

**Review by azabbasi — APPROVED** [review]

**Pull request merged**